xbmc/LangInfo.cpp

```diff
@@ -981,6 +981,15 @@ void CLangInfo::SetCurrentRegion(const std::string& strName)
 
   m_currentRegion->SetGlobalLocale();
 
+  // Check if locale is not affected by #19883
+  int test19883 = ::tolower('i') - ::tolower('I');
+  if (test19883)
+  {
+    CLog::Log(LOGERROR, "locale is affected by #19883 - falling back to default region");
```

Comment:
Should probably log the locale that caused the error and what the default locale is.